### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by DadhichTanishka**
[https://www.loom.com/share/b0d39eddb1f94bd1be1f5d19cf11cbaf?sid=68ae4b26-94b7-4f3e-bc43-54fdd15fe237](https://www.loom.com/share/b0d39eddb1f94bd1be1f5d19cf11cbaf?sid=68ae4b26-94b7-4f3e-bc43-54fdd15fe237)

### CODE REVIEW

**Review by th3c0d3br34ker — COMMENTED** [review]
thank @DadhichTanishka, i have reviewed you loom. 

the explanation was good. however, you have not instantiated the objects and a  demo of the code is also missing.

please add those and let me know.

**Comment by DadhichTanishka** [comment]
hey, I read your review how will I be able to give a demo of the code if I haven't incorporated the constructor concept yet, do you want to see the program running and exiting without any output? also, I have declared the objects so I didn't get what I have to do more can you please tell me more specifically what to do?

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    string EndDate;
+};
+class Salary{
+    private:
+    int employeeID;
+    double salary;
+};
+
+int main(){
+Employee employee1;
```
Comment (th3c0d3br34ker) [inline main.cpp]: the objects were initialized here, but not instantiated.

**Comment by th3c0d3br34ker** [comment]
hey @DadhichTanishka, please book a viva [here](https://calendar.kalvium.com/jainam-desai/30min).

**Comment by DadhichTanishka** [comment]
image

**Review by th3c0d3br34ker — APPROVED** [review]
approved during viva.

please go ahead and merge this PR.

**Pull request merged**